en/getting-started.html

```diff
@@ -1,442 +1,11 @@
-          <a class="footer-link" href="#">
-            <img class="footer-group-logo" src="/assets/external-icons/slack-1.svg" alt="slack" />
-            Slack
-          </a>
-        </div>
-        <div class="footer-description">
-          <a class="footer-link" href="https://www.includecpp.org/">
-            <img class="footer-group-logo" src="/assets/external-icons/discord.svg" alt="discord" />
-            Discord
-          </a>
-        </div>
-        <div class="footer-description">
-          <a class="footer-link" href="https://github.com/microsoft/vcpkg">
-            <img class="footer-group-logo" src="/assets/external-icons/github.svg" alt="github" />
-            Github
-          </a>
-        </div>
-      </div>
-    </div>
-    <div class="footer-right">
-      <div class="footer-info-group-dw">
-        <div class="footer-description">&#169; Microsoft <span id="currentYear"></span></div>
-      </div>
-      <div class="footer-info-group-dw">
-        <div class="footer-description">
-          <a class="footer-link" href="/en/docs/about/privacy.html">Privacy</a>
-        </div>
-      </div>
-      <div class="footer-info-group-privacy">
-        <div class="footer-description">
-          <a style="display: none;" class="managePrivacyChoice footer-link"
-            href="https://aka.ms/yourcaliforniaprivacychoices"><svg role="img" xmlns="http://www.w3.org/2000/svg"
-              width="30px" viewBox="0 0 30 14" xml:space="preserve">
-              <title>California Consumer Privacy Act (CPA) Opt-Out Icon</title>
-              <path d="M7.4 12.8h6.8l3.1-11.6H7.4C4.2 1.2 1.6 3.8 1.6 7s2.6 5.8 5.8 5.8z"
-                style="fill-rule: evenodd; clip-rule: evenodd; fill: #fff"></path>
-              <path
-                d="M22.6 0H7.4c-3.9 0-7 3.1-7 7s3.1 7 7 7h15.2c3.9 0 7-3.1 7-7s-3.2-7-7-7zm-21 7c0-3.2 2.6-5.8 5.8-5.8h9.9l-3.1 11.6H7.4c-3.2 0-5.8-2.6-5.8-5.8z"
-                style="fill-rule: evenodd; clip-rule: evenodd; fill: #06f"></path>
-              <path
-                d="M24.6 4c.2.2.2.6 0 .8L22.5 7l2.2 2.2c.2.2.2.6 0 .8-.2.2-.6.2-.8 0l-2.2-2.2-2.2 2.2c-.2.2-.6.2-.8 0-.2-.2-.2-.6 0-.8L20.8 7l-2.2-2.2c-.2-.2-.2-.6 0-.8.2-.2.6-.2.8 0l2.2 2.2L23.8 4c.2-.2.6-.2.8 0z"
-                style="fill: #fff"></path>
-              <path
-                d="M12.7 4.1c.2.2.3.6.1.8L8.6 9.8c-.1.1-.2.2-.3.2-.2.1-.5.1-.7-.1L5.4 7.7c-.2-.2-.2-.6 0-.8.2-.2.6-.2.8 0L8 8.6l3.8-4.5c.2-.2.6-.2.9 0z"
-                style="fill: #06f"></path>
-            </svg>
-            <span>Your Privacy Choices</span></a>
-        </div>
-      </div>
-    </div>
-  </div>
-  <div style="position: relative"><div class="logo-footer"></div></div>
-  <!--<img class="logo-footer" src="/assets/mark/logo-footer.svg" alt="vcpkg logo" title="vcpkg logo" />-->
-</div>
-</div>
-  </body>
+<head>
+    <meta http-equiv='refresh' content='0;url=https://learn.microsoft.com/en-us/vcpkg/get_started/overview' />
```

Comment:
```suggestion
    <meta http-equiv='refresh' content='0;url=https://learn.microsoft.com/vcpkg/get_started/overview' />
```
We should use unlocalized links so that the user's configured localization applies.

Reply:
hmmm I removed that. For some reason, when I run rebuild.sh it adds it back..